projects/spectator/schematics/src/spectator/index.ts

```diff
@@ -147,3 +147,18 @@ async function _ensurePath(tree: Tree, options: any): Promise<void> {
   options.name = parsedPath.name;
   options.path = parsedPath.path;
 }
+
+/**
+ * create a new object without given keys
+ * @param raw initial object
+ * @param keys list of keys to remove from object
+ */
+function _omit<T extends Record<string, any>>(raw: T, keys: (keyof T)[]): any {
```

Comment:
Thanks for the PR. I think this function is doing the opposite of what you want it to!

Reply:
@benelliott is right. Check your filter logic.